float64 num1
float64 num2
float64 num3
---
float64 avg
